crates/matrix-sdk/src/test_utils/events.rs

```diff
@@ -115,6 +115,12 @@ impl EventBuilder<RoomMessageEventContent> {
             Some(Relation::Reply { in_reply_to: InReplyTo::new(event_id.to_owned()) });
         self
     }
+
+    pub fn replaces(mut self, event_id: &EventId, new_content: RoomMessageEventContent) -> Self {
```

Comment:
Can you add a doc-comment here please?

Reply:
Oh, sure. I though I had handled all the pub fns but I missed this one at least.